### PR TITLE
Page title consistency

### PR DESCRIPTION
**Pull request by mjb44552**
Made the titles and buttons on the my job post, job map and create a job post pages more consistent. The about page changes which I have made in a separate pull request automatically got joined onto this pull requests. I think this is because the changes to that page were already on my machine but not updated to main yet. 

If  we think this will cause any problems comment below.

### CODE REVIEW

File: app/templates/AboutPage.html [hunk 1]
```diff
@@ -52,21 +59,87 @@ <h1>Why Choose Career Compass?</h1>
...
+                        to fostering an inclusive environment where individuals from all backgrounds feel valued, respected, and empowered to thrive. Through our recruitment 
+                        practices and partnerships, we strive to promote diversity, equity, and inclusion in the workplace.
                     </p>
                 </div>
             </div>
         </div>
     </div>
+
+<script>
+document.addEventListener('DOMContentLoaded', function () {
```
Comment (tomve1tch) [inline app/templates/AboutPage.html]: Nice use of event listeners of domcontentloaded to ensure document is parsed before loading in content.

File: app/templates/MyJobPosts.html [hunk 2]
```diff
@@ -2,112 +2,105 @@
...
 
   <!-- Content to be refreshed goes here -->
-<div class="row d-flex justify-content-center align-items-center">
-    <div class="col-md-12">
-        <h1> My Job Posts </h1>
-    </div>
+
+<div class="col-md-6 nonFeedBoxContainer">
+    <h1> My Job Posts </h1>
+    <button class='btn btn-sm submitButton btnSwell nonFeedBoxBtn' onclick="location.href='/JobPost'">
```
Comment (tomve1tch) [inline app/templates/MyJobPosts.html:8]: nice use of consistent styling of buttons

File: app/templates/MyJobPosts.html [hunk 3]
```diff
@@ -2,112 +2,105 @@
...
+        <div class="feedBox feedPageBox feedBoxSwell">
+
+            <!-- words for post title -->
+            <h1> {{post.title}}
+
+                <!-- label for post title and expandable content-->
+                <span>
+
+                    <!-- stylying button with 3 different selectors -->
+                    <button onclick="toggleExpand('expand_btn_{{ loop.index }}')" class = "btn btn-sm btn-outline-info feed_expand_btn" id = "expand_btn_{{ loop.index }}">
```
Comment (tomve1tch) [inline app/templates/MyJobPosts.html]: would be nice to see some styling of this button

File: app/templates/MyJobPosts.html [hunk 4]
```diff
@@ -2,112 +2,105 @@
...
+            <!-- Metadata -->
+            <p><strong> {{ post.date_posted }}</strong> </p>
+            <!-- Expandable content section -->
+            <div class="expandable_content" id= "expandable_content_{{loop.index}}" style="display: none;">
+                <p><strong> Applicants:</strong> </p>
+                {% for applicant in applicant_info %}
+                    {% if applicant[0].post_id == post.id %}
+                        <div style="display: inline-flex;">
+                            <p><b>{{applicant[2].first_name}} {{applicant[2].last_name}}</b></p> 
+                            <p><button class = 'btn-sm' onclick="applicationExpand('applicant_box_{{ loop.index }}')" id = "applicant_btn_{{loop.index}}">+</button></p>
```
Comment (tomve1tch) [inline app/templates/MyJobPosts.html]: would be nice to see more styling of this button to ensure consistency with other buttons in feed page that have a similar purpose to ensure document consistency.

**Review by tomve1tch — APPROVED** [review]
Everything looks good. Would be nice to see some styling of buttons on myjobpost page (expandable buttons) in a future pull request. Otherwise all good :)

**Comment by tomve1tch** [comment]
Pull request has significant conflicts with main branch. Important these conflicts are resolved and reviewed before merging onto main.

**Comment by mjb44552** [comment]
Just merged main into Page Title Consistency branch. Resolved the merge conflicts that resulted and now generating no merge conflicts. Site seems to be running as intended when run off this branch.

**Comment by mjb44552** [comment]
If we are all good with this pull request I am ready to merge it.

**Pull request merged**